app/routes/about.tsx

```diff
@@ -0,0 +1,568 @@
+import {
+  Accordion,
+  AccordionButton,
+  AccordionIcon,
+  AccordionItem,
+  AccordionPanel,
+  Box,
+  ExternalLinkIcon,
+  Flex,
+  Grid,
+  GridItem,
+  Link,
+  Text,
+  VStack,
+} from "@nycplanning/streetscape";
+import { HeaderBar } from "~/components/HeaderBar";
+
+export default function About() {
+  return (
+    <Grid
+      templateColumns={{
+        base: "0 [col-start] 1fr repeat(6, 1fr) 1fr [col-end] 0",
+        md: "1.5dvw [col-start] 1fr repeat(10, 1fr) 1fr [col-end] 1.5dvw",
+        lg: "1.18dvw [col-start] 1fr repeat(10, 1fr) 1fr [col-end] 1.18dvw",
+        xl: "0.86dvw [col-start] 1fr repeat(10, 1fr) 1fr [col-end] 0.86dvw",
+      }}
+      gap={{
+        base: "0 3dvw",
+        md: "0 1.6dvw",
+        lg: "0 1.22dvw",
+        xl: "0 0.94dw",
+      }}
+      gridAutoRows={{
+        base: "7dvh auto auto auto",
+      }}
+      sx={{
+        scrollbarWidth: "none",
+      }}
+    >
+      <HeaderBar />
+      <GridItem
+        gridColumnStart={{
+          base: "2",
+          lg: "3",
+          xl: "4",
+        }}
+        gridColumnEnd={{
+          base: "-2",
+          lg: "10",
+          xl: "9",
+        }}
+        gridRowStart={"2"}
+        gridRowEnd={"3"}
+        pt={8}
+      >
+        <Flex
+          direction={"column"}
+          alignItems={"center"}
+          flexShrink={{ lg: 0 }}
+          sx={{
+            scrollbarWidth: "none",
+          }}
+        >
+          <Text
+            fontSize={"xl"}
+            pb={9}
+            borderBottom={"1px solid"}
+            borderColor={"gray.200"}
+          >
+            The{" "}
+            <Text as="span" fontWeight={"bold"}>
+              Capital Projects Portal (CPP)
+            </Text>{" "}
+            helps New Yorkers explore where and how the City is investing in
+            capital projects and understand community priorities across
+            neighborhoods. CPP brings together information from multiple City
+            sources to{" "}
+            <Text as="span" fontWeight={"bold"}>
+              increase transparency, support planning, and make it easier to
+              explore capital investments and community requests in one place.
+            </Text>
+          </Text>
+          <Text
```

Comment:
Let's have this page use semantic `h1`, `h2`, etc tags. Headings at this level can be `h1` and the accordion headings can be `h2`s. Ideally we should use `<Heading>` for this and use `as` to set the specific element, so `<Heading as="h1">` for example, but if that makes the styling too much of a pain, `<Text as="h1">` will suffice.